oci/spec_opts_linux.go

```diff
@@ -33,15 +33,36 @@ import (
 func WithHostDevices(_ context.Context, _ Client, _ *containers.Container, s *Spec) error {
 	setLinux(s)
 
-	devs, err := getDevices("/dev")
+	devs, err := getDevices("/dev", "")
 	if err != nil {
 		return err
 	}
 	s.Linux.Devices = append(s.Linux.Devices, devs...)
 	return nil
 }
 
-func getDevices(path string) ([]specs.LinuxDevice, error) {
+// WithDevices recurssively adds devices from the past in path.
+func WithDevices(path, containerPath, permissions string) SpecOpts {
```

Comment:
path -> dirPath for explicitness

Reply:
and a typo in `recurssively`

Reply:
I changed this to take either a device path or a dir path.